frame/contracts/src/wasm/runtime.rs

```diff
@@ -813,6 +818,24 @@ define_env!(Env, <E: Ext>,
 		ctx.set_storage(key_ptr, value_ptr, value_len).map(|_| ())
 	},
 
+	// Replace contract code
+	//
+	// # Parameters
+	//
+	// - code_hash_ptr: a pointer to the buffer that contains new contract code
+	//
+	// # Errors
+	//
+	// - requested buffer is not within the bounds of the sandbox memory.
+	// - the buffer contents cannot be decoded as the required type.
+	// - code for specified hash does not exist
+	[__unstable__] seal_set_code_hash(ctx, code_hash_ptr: u32) -> ReturnCode => {
```

Comment:
`ReturnCode` isn't used. This function doesn't return anything but a `Success` of it returns `Ok`.

Reply:
@yarikbratashchuk not sure if this has been addressed either

Reply:
This was addressed. It calls `err_into_return_code` which returns `CodeNotFound` on err.